sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/SubscriptionContainer.cs

```diff
@@ -144,9 +144,18 @@ protected override void Validate(ResourceIdentifier identifier)
         /// <exception cref="ArgumentException"> subscriptionGuid cannot be null or a whitespace. </exception>
         public Response<Subscription> Get(string subscriptionGuid, CancellationToken cancellationToken = default)
         {
-            return new SubscriptionOperations(
-                    new ClientContext(ClientOptions, Credential, BaseUri, Pipeline),
-                    subscriptionGuid).Get(cancellationToken);
+            using var scope = Diagnostics.CreateScope("SubscriptionContainer.Get");
+            scope.Start();
+            try
+            {
+                var response = RestClient.Get(subscriptionGuid, cancellationToken);
+                return Response.FromValue(Converter().Invoke(response.Value), response.GetRawResponse());
```

Comment:
This should just new a Subscription object and lets delete the Converter method on 195.
```suggestion
                return Response.FromValue(new Subscription(this, response.Value), response.GetRawResponse());
```